packages/experimental/Button/MIGRATION.md

```diff
@@ -0,0 +1,52 @@
+# Button Migration
+
+## STATUS: WIP 🚧
+
+This Migration guide is a work in progress and is not yet ready for use.
+
+## Migration from v0 Button
+
+### Component renames
+
+Common buttons now all map to `Button`:
```

Comment:
Except FAB, and MenuButton

Reply:
Reworded to specifically mention just Primary and Stealth buttons, which are the variants that exist on the older button component in FURN.